### ISSUE TITLE
Backbone.Cloudant.View confusion

### ISSUE THREAD

**Issue by rbpasker** [cloudant-labs/backbone.cloudant#7]
this is more of a suggestion than a bug

I have to say that I was somewhat surprised that Backbone.Cloudant.View is really a backbone Collection and not a backbone View

perhaps there's some way to make it less confusing.

**Comment by drsm79** [comment]
Yeah, agreed. There's the naming clash between CouchDB and Backbone. As discussed in IRC, I think I'll switch it to Index, e.g. Index.Collection and Search.Collection, thanks for the comment.

**Comment by intel352** [comment]
Was this resolved? If not, is there a suggested workaround?

**Comment by drsm79** [comment]
It's a naming clash; the word View has different meanings in CouchDB and backbone. I've got the patch to fix it locally, but would like to bump grunt version at the same time. Hopefully will get it done when I'm back from vacation at the end of the week.

**Comment by drsm79** [comment]
@rbpasker @intel352 I think the commit above resolves this - would appreciate your input.

**Comment by rbpasker** [comment]
i've moved off of both backbone and cloudant. sorry I can't help

**Comment by mountainash** [comment]
Both the online test/demo and documentation need to be re-built, as they still refer to the "old" terminology making for a bit of confusion when using the latest dist/... .js files.

**Comment by drsm79** [comment]
Good catch, thanks. Will update ASAP.
